lib/utils/throttle.js

```diff
@@ -0,0 +1,52 @@
+const sleep = require('./sleep.js');
+
+function Throttle () {
```

Comment:
is this your code or you copied or adapted it from somewhere else?

Reply:
Throttle function I've written by myself, sleep function is copied

Reply:
Good, in that case, can you add a docstring explaining how this works (e.g. expected settings) and some basic unit tests showing that it works?

Reply:
Sure, but it will take some time.

Reply:
I have no hurry, thank you for the contribution.